============================= test session starts ==============================
platform linux -- Python 3.10.8, pytest-7.2.1, pluggy-1.0.0
rootdir: /home/runner/InCollege-Software-App
collected 1 item

Tests/testrunner2.py F                                                   [100%]

=================================== FAILURES ===================================
__________________________ test_register_max_entries ___________________________

capsys = <_pytest.capture.CaptureFixture object at 0x7f10687e90f0>
monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x7f10687e9120>

    def test_register_max_entries(capsys, monkeypatch):
    	for i, r in input:
    		print(i, r)
    		test = StringIO(i)
    		monkeypatch.setattr('sys.stdin', test)
>   		assert system.register() == r

Tests/testrunner2.py:18: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <AccountSystem.AccountSystem object at 0x7f10687e8760>

    def register(self):
      success = False
      if self.num_accounts >= MAX_ACCOUNTS:
        input(
          "\nAll permitted accounts have been created, please come back later..."
        )
    
      else:
    
        os.system("clear")
        print(
>         f"Account Registration (# of Accounts Available - {MAX_ACCOUNTS - len(self.num_accounts)}):\n"
        )
E       TypeError: object of type 'int' has no len()

AccountSystem.py:82: TypeError
----------------------------- Captured stdout call -----------------------------
[H[2J[3JD1
I234567*
A
a

 True
=========================== short test summary info ============================
FAILED Tests/testrunner2.py::test_register_max_entries - TypeError: object of...
============================== 1 failed in 0.06s ===============================
